.github/workflows/verify.yml

```diff
@@ -53,7 +53,20 @@ jobs:
     name: lint
     runs-on: ubuntu-latest
     steps:
+      - name: Clear cache
+        uses: actions/cache@v2
+        with:
+          path: |
+            /home/runner/work/_temp
+            /home/runner/work/rudder-server/rudder-server
+          key: ${{ runner.os }}-go-${{ hashFiles('**/go.mod') }}-v1.51.1-${{ hashFiles('**/go.sum') }}
+          restore-keys: |
+            ${{ runner.os }}-go-${{ hashFiles('**/go.mod') }}-v1.51.1-
+            ${{ runner.os }}-go-${{ hashFiles('**/go.mod') }}-
+            ${{ runner.os }}-go-  
       - uses: actions/checkout@v3
+      - name: Delete destination directory
+        run: rm -rf /home/runner/work/rudder-server/rudder-server/ && mkdir /home/runner/work/rudder-server/rudder-server
```

Comment:
why are we changing this?

Reply:
I think this was an attempt to fix the errors of the kind seen here https://github.com/rudderlabs/rudder-server/actions/runs/4449013621/jobs/7812599883?pr=3092 , but with these changes cause another issue in the github action. I am reverting these changes